src/latexify/codegen/function_codegen_test.py

```diff
@@ -744,3 +743,35 @@ def test_use_set_symbols_compare(code: str, latex: str) -> None:
     tree = ast.parse(code).body[0].value
     assert isinstance(tree, ast.Compare)
     assert function_codegen.FunctionCodegen(use_set_symbols=True).visit(tree) == latex
+
+
+def test_matchvalue() -> None:
+    tree = ast.parse(
+        textwrap.dedent(
+        """
+        match x:
+            case 0:
+                return 1
```

Comment:
So this (and the following syntax) should raise the error, according to the comment above.